extensions/multi/src/org/sbml/jsbml/validator/offline/constraints/MultiModelPluginConstraints.java

```diff
@@ -90,10 +93,11 @@ public void addErrorCodesForAttribute(Set<Integer> set, int level,
       func = new UnknownAttributeValidationFunction<MultiModelPlugin>() {
```

Comment:
Here you have an UnknownAttributeValidationFunction which is not useful. You are inheriting from a specific ValidationFunction that is done to check unknown attribute. In this case, you do not need to create an anonymous class, you can directly use DuplicatedElementValidationFunction for the 'func' attribute. If I am not clear in my explanation, you can have a look at MultiSpeciesTypeConstraints, specifically MULTI_20401-3 and MULTI_20406 where we use directly a ValidationFunction implemented elsewhere and we just create a new instance.

Reply:
I'll just have a look at these two rules for clarity.
The ```UnknownAttributeValidationFunction``` was written already (before my work) so I did not change it. 
To resolve this, I guess I just need to delete the anonymous function block and retain the  ```DuplicatedElementValidationFunction```?